lib/alchemy/pg_search/controller_methods.rb

```diff
@@ -49,31 +49,24 @@ def perform_search
       # @return [Array]
       #
       def search_results
-        pages = Alchemy::PgSearch.config[:page_search_scope].pages
-        # Since CanCan cannot (oh the irony) merge +accessible_by+ scope with pg_search scopes,
-        # we need to fake a page object here
-        if can? :show, Alchemy::Page.new(restricted: true, public_on: Date.current)
-          pages.full_text_search(params[:query])
-        else
-          pages.not_restricted.full_text_search(params[:query])
-        end
+        Alchemy::PgSearch.search params[:query], ability: current_ability
```

Comment:
Je mehr man mit anderen Programmiersprachen zu tun hat, desto mehr mag man es doch mit Klammern:

```suggestion
        Alchemy::PgSearch.search(params[:query], ability: current_ability)
```